templates/thank-you.html

```diff
@@ -1,11 +1,38 @@
 {% extends 'layouts/_base.html' %}
+{% from "components/panel/_macro.njk" import onsPanel %}
 
 {% set page_title = _("We\'ve received your answers") + " - " + page_header.title %}
 
 {% block main %}
+  {% if schema_theme == "default" %}
+      <h1 data-qa="submission-successful-title">{{ _("Submission successful") }}</h1>
+      {% call onsPanel({
+            "spacious": true,
+            "type": "success",
+            "classes": "u-mb-s"
+          }) %}
+        <p>
+          {% if metadata.trad_as %}
+              {{ _("Your answers were submitted for <span>%(ru_name)s</span> (%(trading_as_name)s) on %(submitted_date_time)s",
+              ru_name = metadata.ru_name,
+              trading_as_name = metadata.trad_as,
+              submitted_date_time=metadata.submitted_time|format_datetime) }}
+          {% else %}
+              {{ _("Your answers were submitted for <span>%(ru_name)s</span> on %(submitted_date_time)s",
+              ru_name = metadata.ru_name,
+              submitted_date_time=metadata.submitted_time|format_datetime) }}
```

Comment:
We should use `{}` for placeholders in strings rather than `%()s` - it's cleaner and the extracted strings will be consistent with the strings extracted from schemas. This should be:

```
_("Your answers were submitted for <span>{ru_name}</span> ({trading_as_name}) on {submitted_date_time}").format(
              ru_name = metadata.ru_name,
              trading_as_name = metadata.trad_as,
              submitted_date_time=metadata.submitted_time|format_datetime)
```

This should be done for all `%()s` substitutions in this template.

Reply:
All placeholders in this template are now `{}`. Thanks!